components/dashboard/src/whatsnew/MigrationPage.tsx

```diff
@@ -86,19 +83,19 @@ export function MigrationPage() {
                 <div className="mt-24">
                     <OnboardingStep
```

Comment:
This is neat, didn't realize you were able to re-use the `<OnboardingStep>` component here.